### ISSUE TITLE
TypeError: can't access property "type", t[i] is undefined in Next.js

### ISSUE THREAD

**Issue by eIiot** [kommitters/editorjs-undo#156]
### Precheck

- [x] Do a quick search and make sure the bug has not been reported yet.
- [x] Please disclose security vulnerabilities privately at [oss@kommit.co](mailto:oss@kommit.co).
- [x] Finally, be nice and have fun!.

### Description

I'm getting a weird error when using this component with NextJS. The page crashes, but is fixed after reloading, and doesn't appear for a while? It seems to be random. Possibly a SSR issue?

### Environment

* Editor.js version: v2.25.0
* Plugins you use with their versions: DragDrop
* Device, Browser, OS: MacOS in Firefox

### Current behavior

```error
Unhandled Runtime Error

TypeError: can't access property "type", t[i] is undefined
Call Stack
value
node_modules/editorjs-undo/dist/bundle.js (1:6033)
value/<
node_modules/editorjs-undo/dist/bundle.js (1:5490)```

### Expected behavior

The page doesn't crash.

**Comment by psalchow** [comment]
Added PR #175

**Comment by miguelnietoa** [comment]
Hi there! 

Bug solved in the version v2.0.15, thanks @eIiot for reporting, and @psalchow for contributing! ✌🏻
